internal/pkg/service/stream/storage/level/local/diskwriter/network/transport/util.go

```diff
@@ -0,0 +1,33 @@
+package transport
+
+import (
+	"fmt"
+	"io"
+
+	"github.com/hashicorp/yamux"
+
+	"github.com/keboola/keboola-as-code/internal/pkg/log"
+	"github.com/keboola/keboola-as-code/internal/pkg/service/stream/storage/level/local/diskwriter/network"
+)
+
+func sessionKey(session *yamux.Session) string {
+	return session.RemoteAddr().String()
+}
+
+func streamKey(stream *yamux.Stream) string {
+	return fmt.Sprintf(`%s-%d`, stream.RemoteAddr(), stream.StreamID())
+}
+
+func multiplexerConfig(_ log.Logger, config network.Config) *yamux.Config {
+	return &yamux.Config{
+		AcceptBacklog:          config.MaxWaitingStreams,
+		EnableKeepAlive:        true,
+		KeepAliveInterval:      config.KeepAliveInterval,
+		ConnectionWriteTimeout: config.StreamWriteTimeout,
+		MaxStreamWindowSize:    uint32(config.StreamMaxWindow.Bytes()),
+		StreamOpenTimeout:      config.StreamOpenTimeout,
+		StreamCloseTimeout:     config.StreamCloseTimeout,
+		// Logger: log.NewStdErrorLogger(logger.WithComponent("mux")),
```

Comment:
Commented logger.

Reply:
Yes. It logs errors duplicated to those we log ourselves. I will add a comment.

Reply:
https://github.com/keboola/keboola-as-code/pull/1895/commits/582ec368d7cedd05846b1bcfe4368f8b1d9106ce